packages/bitcore-client/bin/multi-sig/wallet-derive-address

```diff
@@ -37,20 +38,22 @@ const parseKeys = keys => {
   }
 };
 
-function getAddress(publicKeyRing, addressIndex, isChange, m, network) {
+function getAddress(publicKeyRing, addressIndex, isChange, m, network, useBech32) {
   const publicKeys = publicKeyRing.map(function(key) {
     const xpub = new bitcoreLib.HDPublicKey(key);
     const changeNum = isChange ? 1 : 0;
     const path = `m/${changeNum}/${addressIndex}`;
     return xpub.derive(path).publicKey;
   });
-  return bitcoreLib.Address.createMultisig(publicKeys, m, network);
+  const nestedWitness = false;
```

Comment:
Why not always use nested witness? It is P2SH so the user will not see the difference, but will save on transaction fees because it's using segwit underneath.